static/js/src/stripe/tests/error-handler.test.js

```diff
@@ -0,0 +1,127 @@
+import { parseForErrorObject } from "../error-handler.js";
+import { contractsErrorObjects, stripeErrorObjects } from "./test-responses.js";
+
+describe("parseForErrorObject", () => {
+  describe("ua-contracts API response handling", () => {
```

Comment:
I think this `describe` block is unnecessary. The `parseForErrorObject` block and the blocks around each test group should be enough.